Poker.BE/Poker.BE.Domain/Game/Round.cs

```diff
@@ -25,69 +26,256 @@ public enum Move
         #region fields
         private ICollection<Player> activeUnfoldedPlayers;
         private Turn currentTurn;
-        private Player dealer;
+        //private Player dealer;        Is dealer needed??
```

Comment:
is he needed?!

![image](https://cloud.githubusercontent.com/assets/9889268/26258901/601656a2-3ccf-11e7-8b13-af43cbc23131.png)


![image](https://cloud.githubusercontent.com/assets/9889268/26258893/525dee3a-3ccf-11e7-9dd0-9b6c7339b4a9.png)